massgov/openmass
# Change config for TFA module to disallow skipping setup

**Description:**
Change the "skip validation" configuration option for TFA from 3 to 0


**Jira:**
https://jira.mass.gov/browse/DP-17994


**To Test:**
Check out this branch on a local environment, where you can use mailhog (just go to mailhog.mass.local in a different tab & voila, you'll see any mail sent by your local environment).
- [ ] Create a new user with the role "editor" or "author" using any email address (no real email will be sent) and make sure to check the "notify user" option
- [ ] Check mailhog.mass.local and copy the link from the new user email
- [ ] Open the link in an incognito tab or a different browser
- [ ] Set a password
- [ ] Confirm you successfully log in and can go to your user profile's security tab to set up TFA - but don't set it up
- [ ] Try to log in as that user again and confirm you aren't able to 
- [ ] Repeat the process, this time with a user that had been blocked but that you unblock





**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/opemass/blob/develop/docs/peer_review_checklist.md)
